src/io/flutter/inspector/InspectorService.java

```diff
@@ -398,12 +398,11 @@ private void onVmServiceReceived(String streamId, Event event) {
     switch (streamId) {
       case VmService.DEBUG_STREAM_ID: {
         if (event.getKind() == EventKind.Inspect) {
-          // Make sure the WidgetInspector on the device switches to show the inspected object
-          // if the inspected object is a Widget or RenderObject.
+          // Assume the inspector in Flutter DevTools or the on device widget
```

Comment:
Assume the inspector in Flutter DevTools or **on the** device widget

Reply:
done